foreman-tasks.gemspec

```
@@ -27,8 +27,8 @@ same resource. It also optionally provides Dynflow infrastructure for using it f
   s.extra_rdoc_files = Dir['README*', 'LICENSE']
 
   s.add_dependency "dynflow", '>= 1.6.0'
+  s.add_dependency 'fugit', '~> 1.8.1'
```

This pins to 1.8.x. Would it be better to use this pin, or do they not respect semver?
```suggestion
  s.add_dependency 'fugit', '>= 1.8.1', '< 2'
```